openapi.yml

```diff
@@ -0,0 +1,337 @@
+openapi: 3.0.1
+info:
+  title: eat-api
+  version: '2.1'
+  description: Simple static API for some (student) food places in Munich.
+servers:
+  - url: https://tum-dev.github.io/eat-api/{language}
+    variables:
+      language:
+        description: For localization the base path may have to be extended by the language.
+        default: ''
+        enum:
+          - ''
+          - 'en/'
+paths:
+  /{canteen_id}/{year}/{week}.json:
+    get:
+      summary: Get a menu for the specified canteen, year and week
+      parameters:
+        - name: canteen_id
+          in: path
+          description: ID of the canteen
+          required: true
+          schema:
+            $ref: '#/components/schemas/Canteen/properties/canteen_id'
+        - name: year
+          in: path
+          description: Year of the menu to get
+          required: true
+          schema:
+            type: integer
+            example: 2022
+        - name: week
+          in: path
+          description: Week of the menu to get with two digits
+          required: true
+          schema:
+            type: string
+            pattern: ^\d{2}$
+            example: '02'
+      responses:
+        '200':
+          description: successful operation
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Week'
+        '404':
+          description: no menu found for specified options
+      tags:
+        - menu
+  /{canteen_id}/combined/combined.json:
+    get:
+      summary: To get all menus for one specific canteen
+      parameters:
+        - name: canteen_id
+          in: path
+          description: ID of the canteen
+          required: true
+          schema:
+            $ref: '#/components/schemas/Canteen/properties/canteen_id'
+      responses:
+        '200':
+          description: successful operation
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/CanteenMenu'
+        '404':
+          description: no menu found for the given canteen
+      tags:
+        - menu
+  /all.json:
+    get:
+      summary: To get all menus for all canteens
+      responses:
+        '200':
+          description: successful operation
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  canteens:
+                    type: array
+                    items:
+                      $ref: '#/components/schemas/CanteenMenu'
+      tags:
+        - menu
+  /all_ref.json:
```

Comment:
I just noticed, that this won't work properly for the english version, that is not updated everyday.
Also, I find it a bit confusing, that we are using the date here for identification, in contrast to the week in the other endpoints (including `all.json`)
How do you think, should we resolve that? Just note that its not working properly in english + mark it as deprecated. Or shall we look into making it work for the english version as well?

Personally I think, the other API endpoints should be sufficient.

Reply:
Yes, this is something we should change and mark it as deprecated. This is something we then can change in v3.
Could you please create an issue for that?

Reply:
I have updated the documentation, and added the deprecation tag to the file